src/connectors/zingmp3.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Connector for Zing MP3 - https://zingmp3.vn
```

Comment:
Should we remove this? Not sure what this adds since the site is in the connectors mapping.